dependencies-path.yaml

```diff
@@ -0,0 +1,54 @@
+- dependency: "etcd"
+  refPaths:
+  - build/root/WORKSPACE
+  - cluster/gce/config-test.sh
+  - cluster/gce/manifests/etcd-empty-dir-cleanup.yaml
+  - cluster/gce/manifests/etcd.manifest
+  - cluster/gce/upgrade-aliases.sh
+  - cluster/images/etcd-empty-dir-cleanup/Makefile
+  - cluster/images/etcd/Makefile
+  - cluster/images/etcd/README.md
+  - cluster/images/etcd/migrate-if-needed.sh
+  - cmd/kubeadm/app/constants/constants.go
+  - cmd/kubeadm/app/constants/constants_test.go
+  - hack/lib/etcd.sh
+  - staging/src/k8s.io/sample-apiserver/artifacts/example/rc.yaml
+  - test/e2e/apimachinery/aggregator.go
+  - test/e2e/framework/nodes_util.go
+  - test/kubemark/start-kubemark.sh
+
+- dependency: "docker"
+  refPaths:
+  - cmd/kubeadm/app/util/system/docker_validator.go
+  - cmd/kubeadm/app/util/system/docker_validator_test.go
+
+- dependency: "golang"
+  refPaths:
+  - build/build-image/cross/Dockerfile
+  - build/build-image/cross/VERSION
+  - build/root/WORKSPACE
+  - test/images/Makefile
+
+- dependency: "cni"
+  refPaths:
+  - build/debian-hyperkube-base/Makefile
+  - build/rpms/kubeadm.spec
+  - build/rpms/kubelet.spec
+  - build/workspace.bzl
+  - cluster/gce/gci/configure.sh
+  - test/e2e_node/remote/utils.go
+
+- dependency: "coreDNS"
+  refPaths:
+  - cluster/addons/dns/coredns/coredns.yaml.base
+  - cluster/addons/dns/coredns/coredns.yaml.in
+  - cluster/addons/dns/coredns/coredns.yaml.sed
```

Comment:
cluster/addons has a lot of addons that are still treated as third-party deps in the release-notes.
not sure if we should track them here, my personal opinion would be to not do so.

Reply:
while I totally agree with you, I'd like to keep them tracked here, until we move the `cluster/` directory out of tree.

Reply:
Agreed. @dims WDYT ?

Reply:
+1 let's start with something and it will start growing

Reply:
Ack. torn on that. but sure, we can start somewhere